uml-right-now/src/pages/pathways.tsx

```diff
@@ -42,16 +42,17 @@ export default function Pathways() {
             flex-col
             justify-start
             items-center
+            h-screen
```

Comment:
Remove this for the time being; we should come up with a separate solution for ensuring that the footer is on the bottom of the screen

Reply:
Done